frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/CheckoutSelectedBranchAction.java

```diff
@@ -28,7 +28,7 @@
  *  <li>{@link DataKeys#KEY_SELECTED_VCS_REPOSITORY}</li>
  * </ul>
  */
-public class CheckOutBranchAction extends DumbAwareAction {
+public class CheckoutSelectedBranchAction extends DumbAwareAction {
```

Comment:
`CheckOut...`? Since that's a (phrasal) verb `(to) check (sth) out` in this context, not a compound noun `checkout`

Reply:
![image](https://user-images.githubusercontent.com/19799111/81393345-8126b500-9120-11ea-9c22-8996d1e283ae.png)

Based on this :arrow_up: image after a short discussion on _Slack_ we decided to `Checkout` rather than `Check Out` (for consistency). 

Resolving related comments :speech_balloon: